File: ishicodes/solutions/2026-06-17/Q83.c
/*
 * Problem: Count vowels and consonants.
 */

#include<stdio.h>
#include<ctype.h>
int main() {
    char s[1000];scanf("%s",s);int v=0,c=0,i=0;while(s[i]){char x=tolower(s[i]);if(isalpha(x)){if(x=='a'||x=='e'||x=='i'||x=='o'||x=='u')v++;else c++;}i++;}printf("Vowels: %d, Consonants: %d",v,c);
    return 0;
}

File: solutions/2026-06-17/Q83.py
# Problem: Count vowels and consonants.

s = input().strip().lower()
v = sum(1 for c in s if c in "aeiou")
c = sum(1 for c in s if c.isalpha() and c not in "aeiou")
print(f"Vowels: {v}, Consonants: {c}")
